### DISCUSSION TITLE
RESOLVED: Laragon vanishes after configuring mail sender.

### DISCUSSION THREAD

**Discussion by Scilly-guy** [leokhoa/laragon#384] [Show and tell]
**RESOLVED**
I just configured Laragon to connect to a brand new gmail account to send emails using mail(). Although the process stated by [the Laragon docs](https://laragon.org/docs/mail-sender.html) to allow less secure app access is deprecated, instead there is a process of creating an app password, which appeared to work and I ran "Test Sending Mail" however I never received the test email. Upon clicking close on the test, Laragon itself closed and I was no longer able to find the .exe anywhere on my computer. I reinstalled Laragon, and repeated everything, this time I got the test email:
  `It means you can use mail() function to send email to Internet easily with Laragon. Let's try:
  
  mail('[leokhoa@gmail.com](mailto:leokhoa@gmail.com)', 'Hi leokhoa', 'I like Mail Sender feature.');
  
  Enjoy programming :)
  Best wishes,
  Laragon.`
But exactly the same happened again, the Laragon.exe vanished.
The laragon.log file has no entries for today/yesterday now as it is just past midnight.

**Comment by rescenic** [comment]
Your computer definitely has viruses.

**Reply by Scilly-guy** [reply]
What my computer has is Anti Virus that does it's job so well that it flags programs sending emails as viruses.

**Comment by leokhoa** [comment]
@Scilly-guy : The email testing feature uses low level TCP/IP communication to internet outside (Gmail) and ShellAPI to parse the console output and displays the result, so it may be the reason. It would be useful if you can ask your AV vendor so they can improve the behaviour.

**Comment by Scilly-guy** [comment]
Heya, I use Avast Anti Virus, as the developer you are probably better placed to take up discussions with AV vendors about whitelists. I have added a personal exception for Laragon but this is bound to be a problem for others also.

    On Thursday, 2 February 2023 at 08:30:03 GMT, Leo Khoa ***@***.***> wrote:  
 
 


@Scilly-guy : The email testing feature uses low level TCP/IP communication to internet outside (Gmail) and ShellAPI to parse the console output and displays the result, so it may be the reason. It would be useful if you can ask your AV vendor so they can improve the behaviour.
